apps/web/src/hooks/useNewDashboardOptIn.ts

```diff
@@ -28,9 +29,11 @@ export function useNewDashboardOptIn() {
     window.location.href = NEW_DASHBOARD_URL || window.location.origin;
   };
 
-  const optIn = () => {
+  const optIn = async () => {
     segment.track('New dashboard opt-in');
-    updateUserOptInStatus(NewDashboardOptInStatusEnum.OPTED_IN);
+    await updateUserOptInStatus(NewDashboardOptInStatusEnum.OPTED_IN);
+    localStorage.setItem('mantine-theme', 'light');
```

Comment:
this needs to be set directly in local storage otherwise when using hook the operation seems to be async and can't be awaited